src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/MockFormTestUtils.cs

```diff
@@ -34,5 +36,27 @@ public static Mock<IFormFile> CreateFormFileMock(string filename, string content
 
             return formFile;
         }
+
+        public static IFormFile CreateFormFileFromResource(string fileName, string? newFileName = null)
+        {
+            var filePath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!,
+                "Resources" + Path.DirectorySeparatorChar + fileName);
+
+            return CreateFormFileFromResourceWithPath(filePath, newFileName ?? fileName);
+        }
```

Comment:
The existing method uses the `fileName` parameter to reference a real file in the `Resources` directory of the project. The optional parameter allowed testing an unconventionally long filename for an existing test case, without having to actually store one in the project.

NB. I didn't create this method, I just moved it out of a test class into a shared utility, and modified it it so it could be reused.

Reply:
Ok thanks for the explanation, I didn't look into where it was being used.